### DISCUSSION TITLE
Updates realtime

### DISCUSSION THREAD

**Discussion by lorypelli** [a-h/templ#993] [General]
I need to restart the server to update the page, is there a way to do it in realtime?

**Comment by pharrisee** [comment]
Generally the templ code is transpiled into Go code which in turn must be built into an executable, so real-time updates are not immediately possible.

That being said, there are tools which might help with the issue you have:

https://templ.guide/commands-and-tools/live-reload
https://templ.guide/commands-and-tools/live-reload-with-other-tools

**Reply by catgoose** [reply]
https://templ.guide/developer-tools/live-reload/

working links
